### DISCUSSION TITLE
How to prevent pop-up prompts when reading to the end of the document？

### DISCUSSION THREAD

**Discussion by wym68** [koreader/koreader#9932] [Q&A] [answered]
**How to prevent pop-up prompts when reading to the end of the document？I didn't want it to appear, but I couldn't find the corresponding settings.**

![image](https://user-images.githubusercontent.com/87969722/208406023-02a29eba-e7b3-4aea-a2a1-5dee5f625da8.png)

**Comment by Frenzie** [comment, marked as answer]
Settings → Document → End of document action

**Reply by wym68** [reply]
Thank you very much. It works!
